finesse/gui/measure_script/script_view.py

```diff
@@ -127,13 +128,10 @@ def _hide_run_dialog(self) -> None:
         del self.run_dialog
 
     def _on_opus_message(
-        self, status: int, text: str, error: Optional[tuple[int, str]]
+        self, status: EM27Status, text: str, error: Optional[tuple[int, str]]
     ) -> None:
         """Increase/decrease the enable counter when the EM27 connects/disconnects."""
-        # According to the manual, these are the possible states the EM27 can be in
-        # while connected
-        connected = 2 <= status <= 5
-
+        connected = status.connected
```

Comment:
This assignment does not seem necessary. Just use `status.connected` directly where relevant.